g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/login/controller/AuthenticationLoginController.java

```diff
@@ -32,10 +31,10 @@
 
 
 
-@RestController("Authorization-authenticationLoginAPI")  
+@RestController("Authentication-authenticationLoginAPI")  
```

Comment:
Does it need to change file name from authorization to authentication?

Reply:
Yes, I have confirm it with Nannan